packages/xrpl/src/Wallet/fundWallet.ts

```diff
@@ -40,6 +40,7 @@ const MAX_ATTEMPTS = 20
  * ```
  *
  * @param this - Client.
+ * @param amount - A custom amount to fund, if undefined or null, the default amount will be 1000.
```

Comment:
I think amount should be added to the `options` object as an optional parameter. (Currently this would actually be a breaking change since it changes the order of parameters, so someone who had code like
`client.fundWallet(wallet)` would not longer match the code signature)

Reply:
changed